### PR TITLE
Set objectives for Hack Camp and each cohort

### PR DESCRIPTION
**Pull request by zachlatta**
This pull request outlines our objective for Hack Camp and our objective for each cohort. These things will be in flux and we'll iterate on our objectives as needed, but this provides a starting point.

/cc @hackedu/hack-camp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,59 @@ first line of code to publishing their first project.
...
+
+##### Cohort 3
+
+The objective of the third cohort is to improve our work from the second cohort.
+We want to do the best job possible satisfying the first part of Hack Camp's
+objective.
+
+##### Cohort 4
+
+The objective of the fourth cohort is to build a set of materials that enables
```
Comment (maxwofford) [inline README.md]: Is the objective to build a set of materials? It seems like building these materials could be done in the off week before the cohort. What will we be doing with the materials during the cohort?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,59 @@ first line of code to publishing their first project.
...
+
+The objective of the second cohort is to only do the first part of the Hack Camp
+objective: creating the experience. Our goal is to create an experience that
+brings people from no coding knowledge to having the ability, motivation, and
+confidence to create coding projects and continue learning new coding concepts
+with others, without the explicit urging or leading of a teacher.
+
+##### Cohort 3
+
+The objective of the third cohort is to improve our work from the second cohort.
```
Comment (maxwofford) [inline README.md:43]: This makes sense, but it might be nice to have reference to documenting our results.

File: README.md [hunk 3]
```diff
@@ -7,3 +7,59 @@ first line of code to publishing their first project.
...
+The objective of Hack Camp is build an set of materials that allows any hackEDU
+team member to reproduce an experience that that brings people from no coding
+knowledge to having the ability, motivation, and confidence to create coding
+projects and continue learning new coding concepts with others, without the
+explicit urging or leading of a teacher.
+
+This can be broken down into two parts: creating the experience and reproducing
+it.
+
+#### Cohorts
```
Comment (maxwofford) [inline README.md:22]: This jumped to 4 indentation levels.

File: README.md (hunk 2 above)
Comment (maxwofford) [inline README.md:43]: My bad, just read #120: `we're not focused on documentation (yet)`

File: README.md (hunk 1 above)
Comment (zachlatta) [inline README.md]: Ah, good catch. Gonna update accordingly.

File: README.md (hunk 2 above)
Comment (zachlatta) [inline README.md:43]: This shouldn't have to be hidden in #120. Agreed that it should be explicit here.

File: README.md (hunk 3 above)
Comment (zachlatta) [inline README.md:22]: Yeah – that was intentional for formatting reasons. Slightly easier to read with a smaller heading in the README. Do you think that's alright?

**Comment by zachlatta** [comment]
@jonleung I'd ask you to do the final review of this, but you're busy running Hack Camp right now. As discussed in the standup this morning, we'll go ahead and merge this without you and then iterate on it as needed.

@MaxWofford can you review this?

**Comment by zachlatta** [comment]
Awesome – I'll squash the commits and merge.

**Comment by jonleung** [comment]
ᐧ
@zachlatta @maxwofford, for the 4th cohort, can the person reproducing the
experience it be Max? I feel that he may be too familiar with how hack camp
works and as such won't accurately represent a club organizer who is not
familiar with hack camp. Perhaps we may want to get someone completely new
who hasn't been to hack camp or worked with us before?

**Comment by zachlatta** [comment]
Yeah, we can certainly look into having someone else run it.

Perhaps we could take one of the participants from a previous cohort, pay them something that's fair (or not), and have them lead it?

**Pull request merged**